.github/workflows/build_hub.yml

```diff
@@ -152,9 +152,6 @@ jobs:
       - name: "Check out changes"
         uses: actions/checkout@v2
 
-      - name: Spin up build containers
```

Comment:
I think this was added by @whytheplatypus in the recent smoke test changes; will they still run as expected?

Reply:
Should be fine because of the other work to not run the `Jooq`, `migrate`, and `flywayMigrate` tasks.

Reply:
...actually maybe it wasn't added to the linting section, so ignore that :-)